test/Serializer/SubjectNormalizerTest.php

```diff
@@ -196,7 +206,7 @@ public function normalizeProvider() : array
             ],
             'snippet' => [
```

Comment:
We should have a `complete snippet` and `minimum` snippet instead of the one, involves moving the `mockSubjectCall()` here like https://github.com/elifesciences/api-sdk-php/blob/c1c6a10bc3118386a300c65a3130827ffcde4d90/test/Serializer/PersonNormalizerTest.php#L203-L258

Reply:
How would the `complete` and `minimum` snippets vary? I thought the snippet was just the `id` and `name`?

https://github.com/elifesciences/api-raml/blob/develop/src/snippets/subject.v1.yaml

Reply:
It's related to the full subject being fetched, currently this doesn't test values not being present.

Reply:
Hope I have understood this correctly. See my recent commit.